library-backend/library-app/src/main/java/org/internship/library/app/persistence/repository/BookJpaRepository.java

```diff
@@ -0,0 +1,13 @@
+package org.internship.library.app.persistence.repository;
+
+import org.internship.library.app.persistence.entity.BookEntity;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import org.springframework.stereotype.Repository;
+
+/**
+ * Repository for BookEntity
+ */
+@Repository
+public interface BookJpaRepository extends PagingAndSortingRepository<BookEntity, String> {
```

Comment:
Rename this one as you've renamed it on the current branch you're working on to be easier for you when you'll merge.